python/mirage/_cython/core.pyx

```diff
@@ -464,6 +833,32 @@ cdef class CyTBGraph:
         t = ctypes.cast(<unsigned long long>ptr, ctypes.c_void_p)
         return STensor(t)
 
+    property grid_dim:
```

Comment:
I was wondering if we can unify `grid_dim` and `block_dim` here. For example, both of them can return a dictionary or a tuple. Let's try not to use list.

Reply:
It seems like block_dim is not used in the graph shown in the paper and actually also not used in the graph visualizer generated🤔. I removed it in current version.